nvflare/private/fed/server/site_security.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Since this is new file added, suggest change the year from 2021 to 2023

Reply:
not sure about this. we should have a standard license header?

Reply:
For the copyright year, I check few open source projects.
For example Google's Tensorflow and NVIDIA's NEMO, both of them are using the year that the file is created.

Reply:
https://github.com/search?q=repo%3Atensorflow%2Ftensorflow+2023&type=code

https://github.com/search?q=repo%3ANVIDIA%2FNeMo+2023&type=code

Reply:
changed to 2023.